### DISCUSSION TITLE
Import nvchad to the starter config

### DISCUSSION THREAD

**Discussion by dmv000** [NvChad/NvChad#3116] [Q&A] [answered]
i just installed nvchad/starter as they told in the docs 
and they also say that nvchad/nvchad the main repo act as a plugin or something like that and with lazy import i can add its config to my starter but i dont know how to do it

this is what it said on the website:

NvChad

This is the main repo for NvChad and its development.
It contains all the plugins, options, mappings, autocmds
All of its modules are in /nvchad

**Comment by siduck** [comment, marked as answer]
require ("nvchad.mappings")

its there in the starter repo code too... u dont have to import anything
